content/tutorial-paychan.md

```diff
@@ -0,0 +1,611 @@
+    - The payee previously redeemed XRP but forgot to record it.
+    - The payee attempted to redeem XRP and recorded the tentative result, but the transaction's final validated result was not the same and the payee neglected to record the final validated result.
+    - The `rippled` server the payee queried has lost sync with the rest of the network or is experiencing an unknown bug. Use the [`server_info` command](reference-rippled.html#server-info) to check the state of the server. (If you can reproduce this situation, please [report an issue](https://github.com/ripple/rippled/issues/).)
+
+After confirming both the signature and the current state of the payment channel, the payee has not yet received the XRP, but is certain that he or she _can_ redeem the XRP as long as the transaction to do so is processed before the channel expires.
+
+
+## 6. Payee provides goods or services.
+
+At this point, the payee can provide goods and services to the payer, knowing that payment is already assured.
+
+For purposes of this tutorial, the payee can give the payer a high-five or equivalent online message as the "goods and services".
+
+
+## 7. Repeat steps 3-6 as desired.
+
+The payer and payee can repeat steps 3 through 6 (creating, transmitting, and verifying claims in exchange for goods and services) as many times and as often as they like without waiting for the Ripple Consensus Ledger itself. The two main limits of this process are:
+
+- The amount of XRP in the payment channel. (If necessary, the payer can send a [PaymentChannelFund transaction][] to increase the total amount of XRP available to the channel.)
+
+- The immutable expiration of the payment channel, if one is set. (The `cancel_after` field in the `account_channels` response shows this.)
+
+
+## 8. When ready, the payee redeems a claim for the authorized amount.
+
+This is the point where the payee finally receives some XRP from the channel.
+
+This is a [PaymentChannelClaim transaction][] with the `Balance`, `Amount`, `Signature`, and `PublicKey` fields provided. Because claim values are cumulative, the payee only needs to redeem the largest (most recent) claim to get the full amount. The payee is not required to redeem the claim for the full amount authorized.
+
+The payee can do this multiple times, to settle partially while still doing business, if desired.
+
+Example of claiming XRP from a channel:
+
+Request:
+
+    POST http://localhost:5005/
+    Content-Type: application/json
+
+    {
+        "method": "submit",
+        "params": [{
+                "secret": "s████████████████████████████",
+                "tx_json": {
+                    "Account": "rf1BiGeXwwQoi8Z2ueFYTEXSwuJYfV2Jpn",
+                    "TransactionType": "PaymentChannelClaim",
+                    "Amount": "1000000",
+                    "Balance": "1000000",
```

Comment:
Probably a naive question, but do the `Balance` and `Amount` need to be the same? Would they only differ if the payee doesn't want to redeem the full amount?